doc/architecture.md

```diff
@@ -207,21 +207,22 @@ the concrete IoT Agent implementations will be to map between the native device
 The following figure offers a graphical example of how a COAP IoT Agent work, ordered from the registration of the
 device to a command update to the device.
 
-![General ](./img/iotAgentLib.png 'Architecture Overview')
+![General ](./img/iotAgentLib.png "Architecture Overview")
 
 ### The `TimeInstant` element
 
 As part of the device to entity mapping process the IoT Agent creates and updates automatically a special timestamp.
 This timestamp is represented as two different properties of the mapped entity::
 
--   An attribute metadata named `TimeInstant` per dynamic attribute mapped, which captures as an ISO8601 timestamp when
-    the associated measurement (represented as attribute value) was observed.
+-   With NGSIv1/NGSI-v2, an attribute metadata named `TimeInstant` (per dynamic attribute mapped, which captures as an
+    ISO8601 timestamp when the associated measurement (represented as attribute value) was observed. With NGSI-LD, the
+    Standard `observedAt` metadata attribute is used instead.
 
--   An entity attribute named `TimeInstant` which captures as an ISO8601 timestamp when the last measurement received
-    from the device was observed.
+-   For NGSIv1/NGSI-v2 only, an additional propert called `TimeInstant` is added to the entity which captures as an
```

Comment:
```suggestion
-   For NGSIv1/NGSI-v2 only, an additional attribute `TimeInstant` is added to the entity which captures as an
```

In NGSIv2 the "property of entity" concept doesn't exist. The equivalent is "attribute".